common/fiovb.c

```diff
@@ -227,6 +227,36 @@ static fiovb_io_result delete_persistent_value(struct fiovb_ops *ops,
 	return rc;
 }
 
+static fiovb_io_result secondary_boot(struct fiovb_ops *ops,
+				      int *secondary_boot,
+				      bool update)
+{
+	fiovb_io_result rc;
+	struct tee_param param[1];
```

Comment:
why does it need to be an array of one element?

Reply:
I can change it to (also addressed your next comment):
```
struct tee_param param = { .attr = TEE_PARAM_ATTR_TYPE_VALUE_INOUT };
```